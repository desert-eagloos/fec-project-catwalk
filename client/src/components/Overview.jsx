import React, { useState, useEffect } from 'react';
import PropTypes from 'prop-types';

function Overview({ product }) {
  const [display, setDisplay] = useState({});

  useEffect(() => {
    setDisplay(product);
  }, []);

  return (
    <div className="overview-component">
      <div className="overview overview-image-gallery-container">
        Image Gallery
        <div className="overview over-image-gallery-default-view">
          IMG PLACEHOLDER
          <button className="overview overview-image-gallery-expanded-view" type="button">ICON HERE</button>
          <div className="overview overview-image-gallery-thumbnails">IMG GALLERY THUMBNAILS</div>
        </div>
      </div>
      <div className="overview overview-product-information-a">
        Production Information
        <div className="overview overview-star-rating">Star Rating</div>
        <div className="overview overview-product-category">Production Category</div>
        <div className="overview overview-product-title">{display.name}</div>
        <div className="overview overview-price">{`$${display.default_price}`}</div>
        <div className="overview overview-social-media">
          Social Media
          <img src="/Assets/Icons/instagram.svg" alt="Instagram" />
        </div>
      </div>
      <div className="overview overview-style-selector">Style Selector</div>
      <div className="overview overview-add-to-cart-container">
        Add to Cart
        <div className="overview overview-size-selector">Size Selector</div>
        <div className="overview overview-quantity-selector">Quantity Selector</div>
        <button type="button" className="overview overview-add-to-cart-button">Add to Cart Button</button>
      </div>
      <div className="overview overview-production-information-b">
        <div className="overview overview-product-description">Product Description</div>
      </div>
    </div>
  );
}

Overview.propTypes = {
  product: PropTypes.shape({
    name: PropTypes.string,
    default_price: PropTypes.string,
  }),
};

Overview.defaultProps = {
  product: {
    name: 'Ernesto\'s Sweatpants',
    default_price: '56.00',
  },
};

export default Overview;
