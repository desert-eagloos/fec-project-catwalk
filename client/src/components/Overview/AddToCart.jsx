import React, { useState, useEffect } from 'react';
// import { Container, Col, Row } from 'react-bootstrap';
import Dropdown from 'react-bootstrap/Dropdown';
import DropdownButton from 'react-bootstrap/DropdownButton';
import Button from 'react-bootstrap/Button';
import Alert from 'react-bootstrap/Alert';
import PropTypes from 'prop-types';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faShoppingCart } from '@fortawesome/free-solid-svg-icons';
// import axios from 'axios';

const _ = require('underscore');
const helpers = require('./OverviewHelpers');

function AddToCart({ cartOptions }) {
  const [selectedSize, setSelectedSize] = useState(null);
  const [selectedQuantity, setSelectedQuantity] = useState(null);
  const [selectedSKU, setSelectedSKU] = useState(null);
  const [sizeDropdownMenuText, setSizeDropdownMenuText] = useState('Select Size');
  const [navigateToSizeSelection, setNavigateToSizeSelection] = useState(false);
  const [showAlert, setShowAlert] = useState(false);
  const [quantityDropdownMenuText, setQuantityDropdownMenuText] = useState('-');
  const [quantityDropdownMenuStatus, setQuantityDropdownMenuDisabled] = useState(true);
  const [quantityOptionsOfSelectedSize, setQuantityOptionsOfSelectedSize] = useState(0);
  const [inStockSizes, setInStockSizes] = useState(
    helpers.filterOutOfStockSizes([]),
  );
  const [outOfStock, setOutOfStock] = useState(true);

  useEffect(() => {
    setInStockSizes(
      helpers.filterOutOfStockSizes(cartOptions),
    );
    setOutOfStock(() => (inStockSizes.length === 0));
    if (selectedSize) {
      if (!outOfStock && helpers.returnSKUForSizeInNewStyle(inStockSizes, selectedSize)) {
        setSelectedSKU(helpers.returnSKUForSizeInNewStyle(inStockSizes, selectedSize));
        setSelectedQuantity(1);
        setQuantityDropdownMenuText('Qty: 1');
      }
    } else {
      setSelectedSize(null);
      setSelectedQuantity(null);
      setSelectedSKU(null);
      setSizeDropdownMenuText('Select Size');
      setQuantityDropdownMenuText('-');
      setQuantityDropdownMenuDisabled(true);
    }
  }, [cartOptions]);

  useEffect(() => {
    setNavigateToSizeSelection(false);
  }, [selectedSize]);

  const renderSizeSelectorMenu = (list) => (
    (list.length > 0) ? (
      <>
        <Alert
          className="mb-0"
          variant="secondary"
          show={showAlert}
        >
          Please Select A Size
        </Alert>
        <DropdownButton
          id="dropdown-basic-button"
          title={sizeDropdownMenuText}
          show={navigateToSizeSelection}
          variant="secondary"
          onClick={() => {
            setNavigateToSizeSelection(!navigateToSizeSelection);
          }}
        >
          {_.map(inStockSizes, (option) => (
            <Dropdown.Item
              key={option.id}
              as="button"
              id={option.id}
              onClick={(event) => {
                setShowAlert(false);
                setSelectedSize(event.target.innerText);
                setSizeDropdownMenuText(`Size: ${event.target.innerText}`);
                setSelectedSKU(event.target.id);
                setQuantityDropdownMenuText('Qty: 1');
                setSelectedQuantity(1);
                setQuantityDropdownMenuDisabled(false);
                const filterCartOptions = (
                  helpers.filterStockOfSelectedSize(cartOptions, event.target.innerText)
                );
                setQuantityOptionsOfSelectedSize(
                  helpers.maxQuantityOptions(filterCartOptions[0].quantity),
                );
              }}
            >
              {option.size}
            </Dropdown.Item>
          ))}
        </DropdownButton>
      </>
    ) : (
      <DropdownButton
        id="dropdown-basic-button"
        title="OUT OF STOCK"
        disabled={outOfStock}
      />
    )
  );

  const renderAddToCartButton = (list) => (
    (list.length > 0) ? (
      <Button
        type="button"
        className="overview overview-add-to-cart-button"
        variant="secondary"
        onClick={(event) => {
          event.preventDefault();
          if (selectedSize === null) {
            setNavigateToSizeSelection(true);
            setShowAlert(true);
          } else {
            helpers.sendAddToCartRequests(selectedQuantity, Number(selectedSKU));
          }
        }}
      >
        <FontAwesomeIcon icon={faShoppingCart} />
        {' Add To Cart'}
      </Button>
    ) : null
  );

  return (
    <div className="overview overview-add-to-cart-container">
      <div className="overview overview-size-selector">
        {renderSizeSelectorMenu(inStockSizes)}
      </div>
      <div className="overview overview-quantity-selector">
        <DropdownButton
          id="dropdown-basic-button"
          title={quantityDropdownMenuText}
          variant="secondary"
          disabled={quantityDropdownMenuStatus}
        >
          {_.map(_.range(1, quantityOptionsOfSelectedSize + 1), (quantity) => (
            <Dropdown.Item
              href="#"
              key={quantity}
              onClick={(event) => {
                setSelectedQuantity(Number(event.target.innerText));
                setQuantityDropdownMenuText(`Qty: ${quantity}`);
                setNavigateToSizeSelection(false);
              }}
            >
              {quantity}
            </Dropdown.Item>
          ))}
        </DropdownButton>
      </div>
      {renderAddToCartButton(inStockSizes)}
    </div>
  );
}

AddToCart.propTypes = {
  cartOptions: PropTypes.arrayOf(PropTypes.shape({
    id: PropTypes.string,
    quantity: PropTypes.number,
    size: PropTypes.string,
  })),
};

AddToCart.defaultProps = {
  cartOptions: [
    { id: '565825', quantity: 45, size: 'XS' },
    { id: '565826', quantity: 27, size: 'S' },
    { id: '565827', quantity: 59, size: 'M' },
    { id: '565828', quantity: 55, size: 'L' },
    { id: '565829', quantity: 51, size: 'XL' },
    { id: '565830', quantity: 10, size: 'XXL' },
  ],
};

export default AddToCart;
