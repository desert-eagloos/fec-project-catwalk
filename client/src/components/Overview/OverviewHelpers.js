const _ = require('underscore');

module.exports = {
  reformatStyleGetResponse: (response) => {
    const newFormat = {
      product_id: response.product_id,
      results: _.map(response.results, (entry) => (
        {
          style_id: entry.style_id,
          name: entry.name,
          original_price: entry.original_price,
          sale_price: entry.sale_price,
          'default?': entry['default?'],
          photos: entry.photos,
          skus: _.map(entry.skus, (value, key) => ({
            sku: key,
            quantity: value.quantity,
            size: value.size,
          })),
        }
      )),
    };
    return newFormat;
  },
  filterCartOptionsBySelectedStyle: (styles, styleName) => {
    const filteredResults = _.find(styles.results, (style) => style.name === styleName);
    const sizeOptionsProp = _.map(filteredResults.skus, (value) => ({
      id: value.sku,
      quantity: value.quantity,
      size: value.size,
    }));
    return sizeOptionsProp;
  },
  findDefaultStyle: (styles) => {
    const defaultStyle = _.find(styles.results, (entry) => entry['default?'] === true);
    return defaultStyle.name;
  },
  findDefaultPrice: (styles) => {
    const defaultStyle = _.find(styles.results, (entry) => entry['default?'] === true);
    return defaultStyle.sale_price ? defaultStyle.sale_price : defaultStyle.original_price;
  },
  filterPriceBySelectedStyle: (styles, styleName) => {
    const filteredResults = _.find(styles.results, (style) => style.name === styleName);
    return filteredResults.sale_price ? filteredResults.sale_price : filteredResults.original_price;
  },
  returnSKUForSizeInNewStyle: (stockList, sizePreviouslySelected) => (
    _.find(stockList, (entry) => entry.size === sizePreviouslySelected).id
  ),
};
