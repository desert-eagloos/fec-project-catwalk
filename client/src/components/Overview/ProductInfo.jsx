import React from 'react';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import PropTypes from 'prop-types';
import {
  faFacebook,
  faInstagram,
  faPinterest,
  faTwitter,
} from '@fortawesome/free-brands-svg-icons';

function ProductInfoA({ product }) {
  return (
    <div className="overview overview-product-information-a">
      <div className="overview overview-star-rating">Star Rating</div>
      <div className="overview overview-product-category">Product Category</div>
      <div className="overview overview-product-title">{product.name}</div>
      <div className="overview overview-price">{`$${product.default_price}`}</div>
      <div className="overview overview-social-media">
        <FontAwesomeIcon icon={faFacebook} size="2x" />
        <FontAwesomeIcon icon={faInstagram} size="2x" />
        <FontAwesomeIcon icon={faPinterest} size="2x" />
        <FontAwesomeIcon icon={faTwitter} size="2x" />
      </div>
    </div>
  );
}

ProductInfoA.propTypes = {
  product: PropTypes.shape({
    id: PropTypes.number,
    name: PropTypes.string,
    default_price: PropTypes.string,
    features: PropTypes.arrayOf(PropTypes.shape({
      feature: PropTypes.string,
      value: PropTypes.string,
    })),
  }),
};

ProductInfoA.defaultProps = {
  product: {
    id: 18201,
    name: 'Ernesto\'s Sweatpants',
    default_price: '56.00',
    features: [{ feature: 'Cut', value: '"Skinny"' }, { feature: 'Cut', value: '"Loose"' }],
  },
};

export default ProductInfoA;
