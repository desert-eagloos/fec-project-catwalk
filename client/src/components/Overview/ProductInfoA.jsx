import React from 'react';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import PropTypes from 'prop-types';
import {
  faFacebook,
  faInstagram,
  faPinterest,
  faTwitter,
} from '@fortawesome/free-brands-svg-icons';

function ProductInfoA({ product, priceByStyle }) {
  if (priceByStyle.salePrice === null) {
    return (
      <div className="overview overview-product-information-a">
        <div className="overview overview-star-rating">Star Rating</div>
        <div className="overview overview-product-category"><h4>{product.category}</h4></div>
        <div className="overview overview-product-title"><h2>{product.name}</h2></div>
        <div className="overview overview-price"><h4>{`$${priceByStyle.originalPrice}`}</h4></div>
        <div className="overview overview-social-media">
          <a href="https://www.facebook.com" rel="noreferrer" target="_blank">
            <FontAwesomeIcon icon={faFacebook} size="2x" />
          </a>
          <a href="https://www.instagram.com" rel="noreferrer" target="_blank">
            <FontAwesomeIcon icon={faInstagram} size="2x" />
          </a>
          <a href="https://www.pinterest.com" rel="noreferrer" target="_blank">
            <FontAwesomeIcon icon={faPinterest} size="2x" />
          </a>
          <a href="https://www.twitter.com" rel="noreferrer" target="_blank">
            <FontAwesomeIcon icon={faTwitter} size="2x" />
          </a>
        </div>
      </div>
    );
  }
  return (
    <div className="overview overview-product-information-a">
      <div className="overview overview-star-rating">Star Rating</div>
      <div className="overview overview-product-category"><h4>{product.category}</h4></div>
      <div className="overview overview-product-title"><h2>{product.name}</h2></div>
      <div className="overview overview-price">
        <h4 className="sale-price-decoration">
          {`$${priceByStyle.salePrice}`}
        </h4>
        <h4 className="original-price-decoration">
          {`$${priceByStyle.originalPrice}`}
        </h4>
      </div>
      <div className="overview overview-social-media">
        <FontAwesomeIcon icon={faFacebook} size="2x" />
        <FontAwesomeIcon icon={faInstagram} size="2x" />
        <FontAwesomeIcon icon={faPinterest} size="2x" />
        <FontAwesomeIcon icon={faTwitter} size="2x" />
      </div>
    </div>
  );
}

ProductInfoA.propTypes = {
  product: PropTypes.shape({
    id: PropTypes.number,
    campus: PropTypes.string,
    slogan: PropTypes.string,
    description: PropTypes.string,
    category: PropTypes.string,
    name: PropTypes.string,
    default_price: PropTypes.string,
    created_at: PropTypes.string,
    updated_at: PropTypes.string,
  }),
  priceByStyle: PropTypes.shape({
    salePrice: PropTypes.string,
    originalPrice: PropTypes.string,
  }),
};

ProductInfoA.defaultProps = {
  product: {
    id: 18201,
    campus: 'hr-bld',
    slogan: 'Odit dolorem nemo id tempora qui.',
    description: 'A sapiente hic. Facilis et sit voluptatem. Ex sunt reiciendis qui ut perferendis qui soluta quod.',
    category: 'Sweatpants',
    name: 'Ernesto Sweatpants',
    default_price: '56.00',
    created_at: '2021-02-23T05:08:00.520Z',
    updated_at: '2021-02-23T05:08:00.520Z',
  },
  priceByStyle: {
    salePrice: null,
    originalPrice: '56.00',
  },
};

export default ProductInfoA;
