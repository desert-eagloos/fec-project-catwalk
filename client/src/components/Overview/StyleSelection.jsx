import React from 'react';
import {
  Container,
  Row,
  Col,
  Image,
  OverlayTrigger,
} from 'react-bootstrap';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faChevronRight, faCheck } from '@fortawesome/free-solid-svg-icons';
import PropTypes from 'prop-types';

const _ = require('underscore');

function StyleSelection({
  styleOptions,
  changeSelectedStyle,
  selectedStyle,
}) {
  return (
    <Container className="mb-4 mt-4">
      <p>
        {'Style '}
        <FontAwesomeIcon icon={faChevronRight} />
        {` ${selectedStyle}`}
      </p>
      <Row lg={4}>
        {_.map(styleOptions.results, (entry) => (
          <Col key={entry.style_id}>
            {/* <OverlayTrigger
              placement="right-end"
              delay={{ show: 250, hide: 400 }}
              trigger="click"
              ref={React.forwardRef()}
              overlay=<FontAwesomeIcon icon={faCheck} />
            > */}
            <Image
              id={entry.name}
              className="style-selection-thumbnails"
              src={entry.photos[0].thumbnail_url}
              roundedCircle
              onClick={(event) => {
                changeSelectedStyle(event.target.id);
              }}
            />
            {/* </OverlayTrigger> */}
          </Col>
        ))}
      </Row>
    </Container>
  );
}

StyleSelection.propTypes = {
  styleOptions: PropTypes.shape({
    product_id: PropTypes.string,
    results: PropTypes.arrayOf(PropTypes.shape({
      style_id: PropTypes.number,
      name: PropTypes.string,
      original_price: PropTypes.string,
      sale_price: PropTypes.string,
      'default?': PropTypes.bool,
      photos: PropTypes.arrayOf(PropTypes.shape({
        thumbnail_url: PropTypes.string,
        url: PropTypes.string,
      })),
      skus: PropTypes.arrayOf(
        PropTypes.shape({
          sku: PropTypes.string,
          quantity: PropTypes.number,
          size: PropTypes.string,
        }),
      ),
    })),
  }),
  changeSelectedStyle: PropTypes.func,
  selectedStyle: PropTypes.string,
};

StyleSelection.defaultProps = {
  styleOptions: {
    product_id: '18201',
    results: [
      {
        style_id: 97715,
        name: 'Green',
        original_price: '56.00',
        sale_price: null,
        'default?': true,
        photos: [
          {
            thumbnail_url: 'https://images.unsplash.com/photo-1482876555840-f31c5ebbff1c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80',
            url: 'https://images.unsplash.com/photo-1515886657613-9f3515b0c78f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=962&q=80',
          },
        ],
        skus: [
          {
            sku: '565825',
            quantity: 45,
            size: 'XS',
          },
          {
            sku: '565826',
            quantity: 27,
            size: 'S',
          },
          {
            sku: '565827',
            quantity: 59,
            size: 'M',
          },
          {
            sku: '565828',
            quantity: 55,
            size: 'L',
          },
          {
            sku: '565829',
            quantity: 51,
            size: 'XL',
          },
          {
            sku: '565830',
            quantity: 10,
            size: 'XXL',
          },
        ],
      },
    ],
  },
  changeSelectedStyle: () => {},
  selectedStyle: 'Green',
};

export default StyleSelection;
