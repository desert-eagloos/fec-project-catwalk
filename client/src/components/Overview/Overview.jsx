import React, { useState, useEffect } from 'react';
import { Container, Col, Row } from 'react-bootstrap';
import PropTypes from 'prop-types';
import axios from 'axios';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import {
  faFacebook,
  faInstagram,
  faPinterest,
  faTwitter,
} from '@fortawesome/free-brands-svg-icons';
import { faShoppingCart } from '@fortawesome/free-solid-svg-icons';

import StyleSelection from './StyleSelection';
import '../../css/overview.css';

const sampleData = require('./sampleStyleData');

function Overview({ product }) {
  const filterStyleOptionProps = (requestResponse) => {
    const filteredResults = requestResponse.results.map((style) => {
      const obj = {
        default: style['default?'],
        name: style.name,
        style_id: style.style_id,
        thumbnail: style.photos[0].thumbnail_url,
      };
      return obj;
    });
    return filteredResults;
  };

  const [styleOptions, setStyleOptions] = useState(filterStyleOptionProps(sampleData.styleGetReq));
  const [selectedStyle, setSelection] = useState();

  const getStylesByProductId = (productId) => {
    axios.get(`/products/${productId}/styles`)
      .then((response) => {
        if (Array.isArray(response.results)) {
          const filterStyleOpts = filterStyleOptionProps(response);
          setStyleOptions(filterStyleOpts);
        }
      })
      .catch();
  };

  // useEffect(() => {
  //   getStylesByProductId(product.id);
  // }, [product]);

  return (
    <Container className="overview-component">
      <Row>
        <Col>
          <div className="overview overview-image-gallery-container">
            Image Gallery
            <div className="overview over-image-gallery-default-view">
              IMG PLACEHOLDER
              <button className="overview overview-image-gallery-expanded-view" type="button">ICON HERE</button>
              <div className="overview overview-image-gallery-thumbnails">IMG GALLERY THUMBNAILS</div>
            </div>
          </div>
        </Col>
        <Col>
          <div className="overview overview-product-information-a">
            <div className="overview overview-star-rating">Star Rating</div>
            <div className="overview overview-product-category">Product Category</div>
            <div className="overview overview-product-title">{product.name}</div>
            <div className="overview overview-price">{`$${product.default_price}`}</div>
            <div className="overview overview-social-media">
              <FontAwesomeIcon icon={faFacebook} size="2x" />
              <FontAwesomeIcon icon={faInstagram} size="2x" />
              <FontAwesomeIcon icon={faPinterest} size="2x" />
              <FontAwesomeIcon icon={faTwitter} size="2x" />
            </div>
          </div>
          <StyleSelection
            styleOpts={styleOptions}
            changeSelection={setSelection}
            selectedStyle={selectedStyle}
          />
          <div className="overview overview-add-to-cart-container">
            Add to Cart
            <div className="overview overview-size-selector">Size Selector</div>
            <div className="overview overview-quantity-selector">Quantity Selector</div>
            <button type="button" className="overview overview-add-to-cart-button">
              <FontAwesomeIcon icon={faShoppingCart} />
              Add To Cart
            </button>
          </div>
        </Col>
      </Row>
      <Row>
        <div className="overview overview-production-information-b">
          <div className="overview overview-product-description">Product Description</div>
        </div>
      </Row>
    </Container>
  );
}

Overview.propTypes = {
  product: PropTypes.shape({
    id: PropTypes.number,
    name: PropTypes.string,
    default_price: PropTypes.string,
    features: PropTypes.arrayOf(PropTypes.shape({
      feature: PropTypes.string,
      value: PropTypes.string,
    })),
  }),
};

Overview.defaultProps = {
  product: {
    id: 18201,
    name: 'Ernesto\'s Sweatpants',
    default_price: '56.00',
    features: [{ feature: 'Cut', value: '"Skinny"' }, { feature: 'Cut', value: '"Loose"' }],
  },
};

export default Overview;
