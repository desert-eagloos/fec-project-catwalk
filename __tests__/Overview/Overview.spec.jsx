/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import Enzyme, { shallow } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import Overview from '../../client/src/components/Overview/Overview';
// import ProductInfoA from '../../client/src/components/Overview/ProductInfoA';

Enzyme.configure({ Adapter: new Adapter() });

describe('Creates Overview Component', () => {
  describe('Overview Snapshot Test', () => {
    test('Overview', () => {
      const wrapper = renderer.create(<Overview />);
      const json = wrapper.toJSON();
      expect(json).toMatchSnapshot();
    });
  });

  describe('isLoading State is functioning', () => {
    test('Finds Empty Tags in Overview When isLoad is true', () => {
      const wrapper = shallow(<Overview />);
      console.log(wrapper.debug({ verbose: true }));
      // console.log(wrapper.state(isLoading));
    });
  });
});
