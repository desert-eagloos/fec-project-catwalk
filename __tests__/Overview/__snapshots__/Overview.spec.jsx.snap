// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Creates Overview Component Overview Snapshot Test Overview 1`] = `null`;
